app/helpers/curation_concern_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module CurationConcernHelper
+  # https://cits.artic.edu/redmine/issues/2183
+  # we needed different titles when uploading single asset vs multiple assets
+  # if it's a GenericWork, it's a single asset, if it's not, it's a BatchUploadItem
+  def custom_h1(curation_concern, action_name)
```

Comment:
Do we always have to pass in the action name since this is only being called from the new.html.erb template?

Reply:
No, we could just use this for new, and do:

```
def custom_h1(curation_concern)
    if curation_concern.is_a?(GenericWork)
      "Create New Asset"
    else
      t("sufia.assets.header")
    end
  end
```